engine/table/src/main/java/io/deephaven/engine/table/impl/util/RowSetShiftCallback.java

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.engine.table.impl.util;
+
+import io.deephaven.engine.rowset.RowSet;
+
+public interface RowSetShiftCallback {
+    void shift(RowSet rowSet, long offset);
```

Comment:
I think this should just be a (JavaDoc'd) method on `WritableColumnSource`, and we should delete the interface.